static/js/components/Editor_test.js

```diff
@@ -0,0 +1,379 @@
+// @flow
+import React from "react"
+import R from "ramda"
+import sinon from "sinon"
+import { mount } from "enzyme"
+import { assert } from "chai"
+import { EditorView } from "prosemirror-view"
+import { schema, defaultMarkdownSerializer } from "prosemirror-markdown"
+import { toggleMark } from "prosemirror-commands"
+import * as pmMarkdown from "prosemirror-markdown"
+import { Provider } from "react-redux"
+
+import AddLinkMenu from "./AddLinkMenu"
+import ConnectedEditor, { Editor, newLinkForm, menuBarManifest } from "./Editor"
+
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { shouldIf } from "../lib/test_utils"
+import * as utilFuncs from "../lib/util"
+import * as pmLibFuncs from "../lib/prosemirror"
```

Comment:
Nit: I think it would be more readable just to import these with the names `util` and `prosemirrorLib` (or `pmLib`) respectively